index.js

```diff
@@ -94,6 +95,8 @@ module.exports = function(connect) {
 			}
 			return store.knex(store.tablename).del()
 			.whereRaw(condition, dateAsISO(store.knex));
+		}).then(function() {
```

Comment:
Should this be finally instead of then? In case the cleanup fails, would we still want to set another timeout for it for the next time?